CoffeeMachine.java

```diff
@@ -0,0 +1,102 @@
+package Lesson_12;
+
+import java.util.Scanner;
+
+public class CoffeeMachine {
+    private String esspresso;
+    private String macchiato;
+    private String cappuccino;
+    private String milk;
+    private String hotChocolate;
+
+    public CoffeeMachine(String esspresso, String macchiato, String cappuccino, String milk, String hotChocolate) {
+        this.esspresso = esspresso;
+        this.macchiato = macchiato;
+        this.cappuccino = cappuccino;
+        this.milk = milk;
+        this.hotChocolate = hotChocolate;
+    }
+
+    private String [] drinkList(){
+        String[] drinkList = {this.esspresso,this.macchiato,this.cappuccino,this.milk,this.hotChocolate};
+        return drinkList;
+    }
+
+    public void getDrink() {
+        System.out.println("Choose your drink:");
+        int n = 0;
+        String[] menu = drinkList();
+        for (int i=0; i< menu.length; i++) {
+            System.out.println(++n + " | " + menu[i]);
+        }
+        makeChoice();
+    }
+
+    private void makeChoice() {
+        Scanner reader = new Scanner(System.in);
+        int answer = reader.nextInt();
+
+        if (answer == 1) {
+            addSugar();
+            System.out.println("Your " + this.esspresso + " is now ready." +  "\n-------------------------");
+        }else if (answer == 2) {
```

Comment:
Same comment from below applies here ( } else instead of }else).
In general, it is good practice to be consistent with these things. Even though they are minor and don't affect functionality, it is better to get accustomed with using them.